client/src/modules/cohort/model/actionTypes.js

```diff
@@ -2,12 +2,12 @@ export const CohortActionTypes = Object.freeze({
   ADD_MEMBER_FAILURE: 'cohort/ADD_MEMBER_FAILURE',
   ADD_MEMBER_REQUEST: 'cohort/ADD_MEMBER_REQUEST',
   ADD_MEMBER_SUCCESS: 'cohort/ADD_MEMBER_SUCCESS',
-  ARCHIVE_FAILURE: 'cohort/ARCHIVE_FAILURE',
-  ARCHIVE_REQUEST: 'cohort/ARCHIVE_REQUEST',
-  ARCHIVE_SUCCESS: 'cohort/ARCHIVE_SUCCESS',
   ADD_OWNER_ROLE_FAILURE: 'cohort/ADD_OWNER_ROLE_FAILURE',
   ADD_OWNER_ROLE_REQUEST: 'cohort/ADD_OWNER_ROLE_REQUEST',
   ADD_OWNER_ROLE_SUCCESS: 'cohort/ADD_OWNER_ROLE_SUCCESS',
+  ARCHIVE_FAILURE: 'cohort/ARCHIVE_FAILURE',
+  ARCHIVE_REQUEST: 'cohort/ARCHIVE_REQUEST',
```

Comment:
Consider: You only change the order here, but if you edited those lines maybe it's good to removed unused request actions because they are not needed now?